### DISCUSSION TITLE
Termux install on Amazon Kindle FX 10?

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#17437] [Q&A] [answered]
Please forgive me if this has been asked before. I did a quick search. I've installed f-droid but says 'no compatible versions.....
I ticked 'include incompatible versions' and ticked the latest version. Install failed, unsurprisingly! Is there any way to install termux on a kindle fire FX?
Thanks.

**Comment by twaik** [comment, marked as answer]
You can install a version from github actions. Let's say [this one](https://github.com/termux/termux-app/actions/runs/5460815053).

**Comment by macerg** [comment]
How about the other termux packages, like termux-boot etc.?

**Reply by twaik** [reply]
Every addon has github repo with Actions enabled. https://github.com/termux/termux-boot/actions

**Reply by macerg** [reply]
Thanks for this. I am very new to this so please understand.
I am not familiar with how 'actions' work. Which one of these should I install? They appear to be zip files, will they be .APK files when extracted?
![Screenshot_20230717-072030](https://github.com/termux/termux-packages/assets/4559048/9054ebb6-48b1-4d26-be7f-a0d7b40194c4)

**Reply by twaik** [reply]
You should choose one artifact correspondiong to CPU architecture your phone has. Or you simply can choose universal one.

**Comment by macerg** [comment]
I installed the universal. The installation was successful but termux crashed on boot.

**Reply by twaik** [reply]
Wdym crashed?

**Comment by macerg** [comment]
Must have been due to wrong version of Android (7) .
Ive installed 5 and it works fine. Thanks.

**Comment by macerg** [comment]
Installed successfully, thanks!

**Comment by macerg** [comment]
Can you please tell me where I can find the android 5 (universal) *. APK file for termux -boot ?

**Reply by twaik** [reply]
Use latest one. It's `minSdkVersion` is 21 which corresponds to Android 5.

**Reply by macerg** [reply]
I'm sorry, but I don't know where to find that.

**Reply by twaik** [reply]
On termux-boot's Actions page...

**Reply by techiegrampa59** [reply]
To OP, How did this work out for you? I'm tempted to try this. Sorry so late to the party.

**Reply by macerg** [reply]
> To OP, How did this work out for you? I'm tempted to try this. Sorry so late to the party.

Worked out just fine!  Go ahead!
